src/database/entities/department.entity.ts

```diff
@@ -0,0 +1,26 @@
+import { Column, DataType, Model, Table } from 'sequelize-typescript';
+
+@Table({
+  tableName: 'departments',
+})
+export class DepartmentEntity extends Model {
+  @Column({
+    type: DataType.INTEGER,
+    primaryKey: true,
+    allowNull: false,
+    autoIncrement: true,
+  })
+  public id: number;
+
+  @Column({
+    type: DataType.STRING,
+    allowNull: false,
+  })
+  public title: string;
+
+  @Column({
+    type: DataType.STRING,
+    allowNull: true,
+  })
+  public description: string;
```

Comment:
```suggestion
  public description: string | null;
```
Тайпскрипту нужно тоже указать, что в этом поле может быть или строка или null, чтобы он тоже это видел